test/Microsoft.Extensions.Logging.Test/LoggerFilterTest.cs

```diff
@@ -0,0 +1,161 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System.Linq;
+using Microsoft.Extensions.Logging.Test;
+using Xunit;
+
+namespace Microsoft.Extensions.Logging
+{
+    public class LoggerFilterTest
+    {
+        [Fact]
+        public void FiltersMessages_OnDefaultLogLevel_BeforeSendingTo_AllRegisteredLoggerProviders()
```

Comment:
I am working on adding a test where messages can further be filtered at individual logger provider levels too.